src/controllers/game-board.controller.js

```diff
@@ -0,0 +1,97 @@
+import ShipModel from '../models/ship.model';
+import GameBoardView from '../views/game-board.view';
+import AppService from '../app.service';
+
+const GameBoardController = (pType) => {
+  const boardArray = []; // [0|1|Ship]
+  const ships = [];
+  const playerType = pType; // [AI|Human]
+  const appService = AppService();
+
+  // Init a 10x10 board filled with zeros.
```

Comment:
**Kindly remove commented code inside this file, you can always use git history to access previous code right**:slightly_smiling_face: